autogen/json2class.cpp

```diff
@@ -1548,6 +1593,47 @@ void read(const std::string &file, nlohmann::json &jdoc)
       throw std::exception{};
    }
    ifs >> jdoc;
+
+   (void)firsttime; // so no "unused" warning
+#if 0
```

Comment:
I'm not a fan of compiler directives for including pieces of code. You should consider putting this in an "information" function instead.

Reply:
OK, will-do.